jetty-start/src/main/java/org/eclipse/jetty/start/BaseBuilder.java

```diff
@@ -169,35 +172,70 @@ else if (!licensing.acknowledgeLicenses())
         Path startd = getBaseHome().getBasePath("start.d");
         Path startini = getBaseHome().getBasePath("start.ini");
 
-        if (startArgs.isCreateStartd() && !Files.exists(startd))
+        if (startArgs.isCreateStartIni())
         {
-            if (FS.ensureDirectoryExists(startd))
+            if (!Files.exists(startini))
             {
-                StartLog.log("MKDIR", baseHome.toShortForm(startd));
+                StartLog.info("create " + baseHome.toShortForm(startini));
```

Comment:
This should be `mkdir` for consistency with other logs, not `create`.

Reply:
but it is not a directory!  "touch" ?

Reply:
Changed from `startd` to `startini` ?

Reply:
@joakime exectly because the option is now `--create-start-ini` and `--create-startd` is a noop

Reply:
Note that `--create-start-ini`  takes all the files in `start.d` and concats them into a start.ini and then removes `start.d`